src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -802,6 +802,72 @@ static int _xccdf_item_recursive_gather_selected_rules(struct xccdf_policy *poli
 	return ret;
 }
 
+static void _trim_trailing_whitespace(char *str, size_t str_len)
+{
+	char *last_char = str + str_len - 1;
+	while (isspace(*last_char)) {
+		*last_char = '\0';
+		last_char--;
+	}
+}
+
+/* Handles multiline strings in profile title and description.
+ * Puts a '#' at the beginning of each line.
+ * Also removes trailing and leading whitespaces on each line.
+ */
+static char *_comment_multiline_text(char *text)
+{
+	if (text == NULL) {
+		return oscap_strdup("Not available");
+	}
+	const char *filler = "\n# ";
+	size_t buffer_size = strlen(text) + 1; // +1 for terminating '\0'
+	char *buffer = malloc(buffer_size);
+	char *saveptr;
+	size_t filler_len = strlen(filler);
+	size_t result_len = 0;
+	bool first = true;
+	char *str = text;
+	while (true) {
+		char *token = oscap_strtok_r(str, "\n", &saveptr);
+		if (token == NULL) {
+			break;
+		}
+		/* Strip leading whitespace */
+		while (isspace(*token)) {
+			token++;
+		}
+		size_t token_len = strlen(token);
+		if (token_len > 0) {
+			/* Strip trailing whitespace */
```

Comment:
Pleare remove that comment - it is not needed, as you have introduced the `_trim_trailing_whitespace` function, so it is now obvious what is going on.